### PR TITLE
7 migrate UI to flowbite

### PR DESCRIPTION
**Pull request by llettieri**

### CODE REVIEW

**Review by nilshindermann — CHANGES_REQUESTED** [review]
Some feedback / things to do:

- [x] Header links (brand + search) are non-`next/link` components
- [x] All card links are non-`next/link` components
- [x] Movie/Series title is always a link, even if it's href is empty
- [x] `Warning: Each child in a list should have a unique "key" prop.`
      I think this comes from `CreditsTable.tsx`

Please also fix the bugs you found yourself 😄

File: components/Footer.tsx [hunk 1]
```diff
@@ -1,15 +1,20 @@
...
+        <FBFooter container className="rounded-none bg-neutral *:text-white">
+            <FooterCopyright
+                href="https://lore-le.ch"
+                by="Lettieri Lorenzo"
+                year={currentYear}
+            />
+            {/* <FooterLinkGroup>
+                <FooterLink href="#">Imprint</FooterLink>
+                <FooterLink href="#">Privacy Policy</FooterLink>
+            </FooterLinkGroup>*/}
```
[the remark this answers is not in the document]
Comment (llettieri) [reply components/Footer.tsx:17]: WIP: see ticket https://github.com/llettieri/movies-series-lookup/issues/5

File: .eslintrc.js [hunk 2]
```diff
@@ -65,5 +65,6 @@ module.exports = {
         camelcase: ['warn'],
         '@typescript-eslint/explicit-function-return-type': ['warn'],
         '@typescript-eslint/method-signature-style': ['error']
-    }
+    },
+    ignorePatterns: ['manifest.ts']
```
Comment (nilshindermann) [inline .eslintrc.js:69]: Why is this needed?

File: components/CreditsTable.tsx [hunk 3]
```diff
@@ -82,7 +82,9 @@ export const CreditsTable = ({
                         </h1>
                         <ul className="flex flex-col justify-center gap-4 md:justify-start">
                             {cast.map((p) => (
-                                <li key={p.id}>
+                                <li
+                                    key={`${p.id}_${Math.round(Math.random() * 1000)}`}
```
Comment (nilshindermann) [inline components/CreditsTable.tsx:86]: [I think this is bad practice](https://react.dev/learn/rendering-lists#where-to-get-your-key:~:text=Keys%20must%20not%20change%20or%20that%20defeats%20their%20purpose!%20Don%E2%80%99t%20generate%20them%20while%20rendering.)

**Review by nilshindermann — COMMENTED** [review]
Almost there 😉

**Review by nilshindermann — APPROVED** [review]
nice work! ❤️

**Pull request merged**